src/actions/getDomainMetadata.ts

```diff
@@ -0,0 +1,21 @@
+import { Registrar } from "../contracts/types";
+import { DomainMetadata } from "../types"
+import { makeApiCall } from "../api/actions/helpers";
+
+export const getDomainMetadata = async (domainId: string, registrar: Registrar) => {
+  const metadataUri = await registrar.tokenURI(domainId);
+
+  // DomainURI is always one of the two below
+  // https://ipfs.fleek.co/ipfs/Qm...
+  // ipfs://Qm...
+  let metadata: DomainMetadata;
+  if (!metadataUri.includes("fleek")) {
+    const qmHash = metadataUri.split("//")[1];
+    const formattedUri = `https://ipfs.io/ipfs/${qmHash}`;
```

Comment:
Would be nice if we could easily configure which IPFS gateway to use. Could we maybe pass in an `ipfsGatewayUri` to this function?